ir/memory.cpp

```diff
@@ -1523,20 +1573,26 @@ StateValue Memory::load(const Pointer &ptr, const Type &type, set<expr> &undef,
     for (auto &p : all_leaf_ptrs(*this, val.value)) {
       auto islocal = p.isLocal();
       auto bid = p.getShortBid();
-      if (!islocal.isTrue() && !bid.isConst()) {
```

Comment:
I think there's no change here anymore. If not, can you pls revert to the old so the diff becomes smaller?